modules/odc-grouping-multiple-components.adoc

```diff
@@ -11,20 +11,25 @@ The following procedure adds a MariaDB database service to an existing applicati
 
 .Prerequisites
 
-* Ensure that you have created and deployed a Node.js application on {product-title} using the *Developer* perspective.
+* Ensure that you have created and deployed minimum two or more components on {product-title} using the *Developer* perspective.
 
 .Procedure
 
-. Create and deploy the MariaDB service to your project:
-.. In the *Developer* perspective, navigate to the *+Add* view and select the *From Catalog* option to view all available services in the *Developer Catalog*.
-.. Select *Databases* to list all the database services and then click *MariaDB* option to see the details for the service.
-.. Click *Instantiate Template* to see an automatically populated template with details for the MariaDB service, and then click *Create* to create and view the MariaDB service in the *Topology* view.
+. To add a service to the existing application group, press kbd:[Shift]+ drag it to the existing application group
 
-. To add the MariaDB service to the existing application group, press kbd:[Shift]+ drag the *mariadb* pod to the application group that contains the Node.Js component. The MariaDB service is added to the existing application group.
-
-. Click the MariaDB service node and confirm that the `app.kubernetes.io/part-of=<application-name>` label is in the *Labels* section in the *Overview* Panel. Adding a component to an application group automatically adds the required labels to the component.
+. Dragging a component and adding it to an application group automatically adds the required labels to the component.
```

Comment:
This seems like a result, not a step. Perhaps you can add the key result to the previous step.

Reply:
Does the word "automatically" add meaningful information to this or any other step? With software, if you perform an action, it has one or more results.